.github/workflows/pr-merged-tag.yml

```
@@ -0,0 +1,132 @@
+            fi
+          done
+
+          # 调试输出
+          echo "最终 TAGS_JSON: $TAGS_JSON"
+          echo "HAS_CHANGES: $HAS_CHANGES"
+
+          # 输出标签JSON和变更状态
+          echo "tags_json=$TAGS_JSON" >> $GITHUB_OUTPUT
+          echo "has_changes=$HAS_CHANGES" >> $GITHUB_OUTPUT
+
+      - uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)